### PR TITLE
chore: add tags to chapter

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

clear the first task in #1339

<details>

<summary>Chapter page Looks</summary> 

image

</details>

<details>

<summary>Edit Chapter Page Looks</summary>

image

</details>

### CODE REVIEW

**Comment by Sboonny** [comment]
if I am not mistaken, currently the issue is there is no textarea or tags shown in the homepage, I am planning to create another PR for these, because this has already changed 9 lines and don't want to change more for QOL

**Comment by Sboonny** [comment]
Chapter page Looks 
image

**Comment by Sboonny** [comment]
2nd attempt to make it array

File: server/prisma/generator/factories/chapters.factory.ts [hunk 1]
```diff
@@ -12,7 +12,21 @@ const createChapters = async (userId: number): Promise<number[]> => {
...
+      'Dog Adoring',
+      'SNAKES',
+      'DevOps',
+      'Consulting',
+      'Hang Out',
+      'Security',
+    ];
+    const randomTag = Math.floor(Math.random() * tagNames.length);
+
+    const tag = tagNames[randomTag];
```
Comment (ojeytonwilliams) [inline server/prisma/generator/factories/chapters.factory.ts:29]: You can use the random.ts library if you like. Doesn't really matter, but it exists if you're interested.

**Review by ojeytonwilliams — COMMENTED** [review]
I like the look of the tags, but, similarly to https://github.com/freeCodeCamp/chapter/pull/1341 , why only one?

File: server/prisma/generator/factories/chapters.factory.ts (hunk 1 above)
Comment (ojeytonwilliams) [reply server/prisma/generator/factories/chapters.factory.ts:29]: > 2nd attempt to make it array

Oh, sorry, didn't see that!

Comment (Sboonny) [reply server/prisma/generator/factories/chapters.factory.ts:29]: Nah, as always any tips and guidance are welcomed. I will check it out and note it when I display tags in the card

**Pull request closed without merging**